manual/parts/vspace.tex

```diff
@@ -169,7 +169,8 @@ \subsection{Page}
 are specified with an argument of type \texttt{seL4\_CapRights} given to the mapping invocation. If
 the capability does not have sufficient permissions to authorise the given mapping, the mapping
 permissions are silently downgraded. Specific mapping permissions are dependent on the architecture
-and are documented in the \autoref{sec:api_reference} for each function.
+and are documented in the \autoref{sec:api_reference} for each function. On all architectures,
```

Comment:
Maybe weaken this a bit. 
```suggestion
and are documented in the \autoref{sec:api_reference} for each function. On all currently supported architectures,
```

Reply:
I don't think that weakening is necessary, we're clearly not talking about all architectures in the world, just the ones seL4 runs on. Fewer words are usually clearer.